src/closure.rs

```diff
@@ -82,6 +82,7 @@ impl Closure {
         from_glib_none(closure)
     }
 
+    #[allow(clippy::redundant_closure)]
     pub fn invoke(&self, values: &[&ToValue]) -> Option<Value> {
```

Comment:
What does this mean exactly, what redudant closure and why can't we fix that?

Reply:
It wan't change `.map(|v| v.to_value())` to function but if I change it, then will be error.
```
error[E0277]: the trait bound `dyn value::ToValue: value::SetValue` is not satisfied
  --> D:/eap/rust/0/glib\src\closure.rs:98:22
   |
98 |                 .map(ToValue::to_value)
   |                      ^^^^^^^^^^^^^^^^^ the trait `value::SetValue` is not implemented for `dyn value::ToValue`
   |
   = note: required because of the requirements on the impl of `value::SetValue` for `&dyn value::ToValue`
   = note: required because of the requirements on the impl of `value::ToValue` for `&dyn value::ToValue`
note: required by `value::ToValue::to_value`
  --> D:/eap/rust/0/glib\src\value.rs:594:5
   |
594|     fn to_value(&self) -> Value;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Same problem for second `redundant_closure`
```
error[E0277]: the trait bound `std::vec::Vec<std::string::String>: translate::ToGlibPtr<'_, _>` is not satisfied
   --> D:/eap/rust/0/glib\src\value.rs:871:46
    |
871 |         let ptr: *mut *mut c_char = this.map(ToGlibPtr::to_glib_full).unwrap_or(ptr::null_mut());
    |                                              ^^^^^^^^^^^^^^^^^^^^^^^ the trait `translate::ToGlibPtr<'_, _>` is not implemented for `std::vec::Vec<std::string::String>`
    |
note: required by `translate::ToGlibPtr::to_glib_full`
   --> D:/eap/rust/0/glib\src\translate.rs:259:5
    |
259 |     fn to_glib_full(&self) -> P {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0277]: the trait bound `std::vec::Vec<std::string::String>: translate::ToGlibPtr<'_, _>` is not satisfied
   --> D:/eap/rust/0/glib\src\value.rs:871:37
    |
871 |         let ptr: *mut *mut c_char = this.map(ToGlibPtr::to_glib_full).unwrap_or(ptr::null_mut());
    |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `translate::ToGlibPtr<'_, _>` is not implemented for `std::vec::Vec<std::string::String>`
```
```

Reply:
Does this need a `as_ref()` maybe? But fine with me

Reply:
Not works too.

Reply:
:man_shrugging: